app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,84 @@
+class ReviewsController < ApplicationController
+  before_action :set_review, only: [:show, :edit, :update, :destroy]
+  before_filter :load_resturant
+
+  # GET /reviews
+  # GET /reviews.json
+  def index
+    @reviews = @resturant.reviews
+  end
+
+  # GET /reviews/1
+  # GET /reviews/1.json
+  def show
+    @review = @resturant.reviews.find(params[:id])
+  end
+
+  # GET /reviews/new
+  def new
+    @review = @resturant.reviews.new
+  end
+
+  # GET /reviews/1/edit
+  def edit
```

Comment:
Not necessary - better not add capabilities if we don't need them (No product requirement to edit review)